recipes/pvlib-python/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.3.2" %}
+
+package:
+  name: pvlib-python
+  version: {{ version }}
+
+source:
+  fn: pvlib-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/58/3d/8431a36635ad7a06c2fa00c53769074279364bea74ab87f99389b8a90a45/pvlib-{{ version }}.tar.gz
```

Comment:
Please change to this ( https://pypi.io/packages/source/p/pvlib/pvlib-0.3.2.tar.gz ). Unfortunately, PyPI has added these ugly hashes to the URL and they seem unwilling to fix them. [pypi.io]( https://pypi.io/ ) just redirects to the URL with hashes.